hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/FederationClientInterceptor.java

```diff
@@ -1998,4 +2088,5 @@ protected int getNumMaxThreads(Configuration conf) {
   public void setNumSubmitRetries(int numSubmitRetries) {
     this.numSubmitRetries = numSubmitRetries;
   }
+
```

Comment:
Avoid

Reply:
Thank you very much for reviewing the code, I will fix it.